docs/.vuepress/config.js

```diff
@@ -269,6 +269,7 @@ module.exports = {
           {
             title: 'Case studies',
             children: [
+              ['/concepts/comparisons', 'Comparisons'],
```

Comment:
This isn't a case study. I would consider putting this in the IPFS 101 category.